Makefile

```diff
@@ -99,14 +101,14 @@ build_python_sdk:: gen_python_sdk
 # Java SDK
 bin/pulumi-java-gen::
 	mkdir -p bin/
-	$(shell pulumictl download-binary -n pulumi-language-java -v $(JAVA_GEN_VERSION) -r pulumi/pulumi-java)
+	pulumictl download-binary -n pulumi-language-java -v $(JAVA_GEN_VERSION) -r pulumi/pulumi-java
 
 gen_java_sdk:: bin/pulumi-java-gen
 	rm -rf sdk/java
 	$(WORKING_DIR)/bin/$(JAVA_GEN) generate --schema ${SCHEMA_PATH} --out sdk/java --build gradle-nexus
 
 build_java_sdk:: PACKAGE_VERSION := $(VERSION)
-build_java_sdk:: gen_java_sdk
+build_java_sdk::
```

Comment:
Why are we removing this dependency? It feels like we do want to have a java sdk before we build it.

Reply:
I added the dependency myself, in fact, but had to back it out. See the [commit message](https://github.com/pulumi/pulumi-aws-apigateway/commit/9bcae2e7e6ec97789d478e2e69b3106b4f3b2e73).

Reply:
Given that this *was* working (since it got checked in) and isn't working now, I think we should understand why. The failed workflow run linked in the commit message makes it look like there is a dependency issue in the Makefile.